torch/onnx/utils.py

```diff
@@ -1471,7 +1471,14 @@ def _get_module_attributes(module):
     annotations = typing.get_type_hints(type(module))
     base_m_annotations = typing.get_type_hints(torch.nn.Module)
     [annotations.pop(k, None) for k in base_m_annotations]
-    return {k: getattr(module, k) for k in annotations}
+    attrs = {}
+    for k in annotations:
+        try:
+            v = getattr(module, k)
+            attrs[k] = v
+        except:
+            continue
+    return attrs
```

Comment:
Perhaps the whole thing can just be

```suggestion
    return {k: getattr(module, k) for k in annotations if hasattr(module, k)}
```

?

Reply:
Agreed that this is better. The initial implementation was just for unblocking the export locally. For a better implementation, the above comments suggest logging the skipped attribute. As such, it will be easier to log the attribute by modifying the initial implementation.